libpkgconf/queue.c

```diff
@@ -249,23 +264,24 @@ pkgconf_queue_verify(pkgconf_client_t *client, pkgconf_pkg_t *world, pkgconf_lis
 		return result;
 	}
 
+	/* determine identifiers for topological sort */
```

Comment:
IIUC, this is not the topological order, this is the node visited order.
The topological order is the "reversed node exited" order.

Reply:
What I'm thinking about is something like this:
https://github.com/qazwsxedcrfvtg14/pkgconf/commit/85daab772a356597570e076aead44adfdbacbedc